lib/nodes/modbus.cpp

```diff
@@ -904,6 +904,17 @@ std::vector<int> ModbusNode::getPollFDs()
 	return { read_task.getFD() };
 }
 
+std::vector<int> ModbusNode::getNetemFDs()
+{
+	if (modbus_context != nullptr && std::get_if<Tcp>(&connection_settings) != nullptr)  {
```

Comment:
You should use [`std::holds_alternative`](https://en.cppreference.com/w/cpp/utility/variant/holds_alternative) if you're only interested in the current variant. `std::get_if` should only be used if you need access to the contents of the variant.